crypto/aes/aes_ige.c

```diff
@@ -38,15 +39,31 @@ void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
                      size_t length, const AES_KEY *key,
                      unsigned char *ivec, const int enc)
 {
+    /* Ignore the return value since we are void */
+    int ret =AES_ige_encrypt_ex(in, out, length, key, ivec, enc);
+
+    assert(ret);
```

Comment:
Personally, I would prefer if `AES_ige_encrypt_ex(in, out, length, key, ivec, enc)` was directly the `assert` argument.